src/store/helpers/index.js

```diff
@@ -16,13 +16,13 @@ export const createRSQLQuery = (state) => transformToRSQL({
   operator: 'AND',
   operands: flatten([
     createInQuery('country', state.filters.selections.country || []),
-    createInQuery('materials', state.filters.selections.materials || []),
-    createInQuery('type', state.filters.selections.type || []),
-    createInQuery('data_categories', state.filters.selections.dataType || []),
-    createInQuery('diagnosis_available.code', state.filters.selections.diagnosis_available || []),
-    createInQuery('id', state.collectionIdsWithSelectedQuality),
+    createQueryWithSatisfyAll(state.filters.selections.materials, 'materials', state.filters.satisfyAll.materials),
```

Comment:
name it createQuery, the name and the returning type do not match, because it doesn't create a query with satisfy all, all the time

Reply:
Resolved in 7e86e3f